packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Checkmarks.Example.tsx

```diff
@@ -93,7 +93,11 @@ export class ContextualMenuCheckmarksExample extends React.Component<any, IConte
                     }
                   ],
                 },
-                name: 'New'
+                name: 'Split Button',
```

Comment:
For the sake of website documentation, could you either give this example it's own ExampleCard in the ContextualMenuPage or change the title in the Checkmarks ExampleCard to something like
title='ContextualMenu with checkable menu items and split button
Thanks!

Reply:
Will do! I will change the name of the example card